spec/chapters/controllers.tex

```diff
@@ -94,6 +94,9 @@ \section{Controllers}
 of fields and properties is unrestricted and fully compatible with \jaxrs\ ---modulo
 the restrictions explained in Section \ref{controller_instances}.
 
+Controller methods handle a HTTP request directly. Sub-resource locators as described
+in the \jaxrs\ Specification \cite{jaxrs} are not supported by \mvc\.
```

Comment:
There is a minor problem with this line. You have to replace replace `\mvc\.` with `\mvc.` Currently the PDF isn't generated at all due to this. Latex is a beast. :smile: